x-pack/plugins/maps/public/classes/sources/ems_file_source/ems_file_source.tsx

```diff
@@ -135,7 +135,7 @@ export class EMSFileSource extends AbstractVectorSource implements IEmsFileSourc
         meta: {},
       };
     } catch (error) {
-      throw new Error(`${getErrorInfo(this._descriptor.id)} - ${error.message}`);
+      throw new Error(`${getErrorInfo(this._descriptor.id)}`);
```

Comment:
This can just be `throw new Error(getErrorInfo(this._descriptor.id));`

Reply:
yes, incorporated the feedback.